components/org.wso2.carbon.consent.mgt.core/src/main/java/org/wso2/carbon/consent/mgt/core/ConsentManagerImpl.java

```diff
@@ -671,9 +738,31 @@ public void deleteReceipt(String receiptId) throws ConsentManagementException {
         }
     }
 
+    /**
+     * This API is used to delete existing receipts by tenant id.
+     *
+     * @param tenantId Id of the tenant.
+     * @throws ConsentManagementException Consent Management Exception.
+     */
+    @Override
+    public void deleteReceipts(int tenantId) throws ConsentManagementException {
+
+        if (tenantId <= 0) {
+            if (log.isDebugEnabled()) {
+                log.debug("Tenant id is not found in the request or invalid tenant id");
```

Comment:
We can just say, invalid tenant id: {tenantId}

Reply:
Fix in other applicable places

Reply:
👍